internal/website/gatherexamples/gatherexamples.go

```diff
@@ -104,10 +110,15 @@ const gatherLoadMode packages.LoadMode = packages.NeedName |
 // the example should be included in the output.
 const signifierCommentPrefix = "// This example is used in"
 
+type example struct {
+	Imports string `json:"imports"`
```

Comment:
Yeah, end context is stuff like: https://github.com/google/go-cloud/commit/16178db8fd96bcab7f3119e7a45350ab99e5ec7a#diff-e9d30c7bdf1ac89962ef46f0c16829f7R57

I found it useful to include or not include imports depending on context of the surrounding docs. Otherwise, the imports take over the example where you only care about the action, not the setup.

Reply:
Got it. It might be useful to add a comment explaining this.

Reply:
Added more to the package-level commentary.